tests/unit/parsers/test.manifestjson.js

```diff
@@ -42,6 +42,21 @@ describe('ManifestJSONParser', () => {
     expect(metadata.version).toEqual(null);
   });
 
+  it('should see browser_specific_settings as alias of applications', () => {
+    const addonLinter = new Linter({ _: ['bar'] });
+    const json = validManifestJSON({
+      applications: undefined,
+      browser_specific_settings: {gecko: { id: 'foo@baa' } },
```

Comment:
@eviljeff this is [the line that is currently triggering the eslint failure on travis](https://travis-ci.org/mozilla/addons-linter/jobs/457538319#L1133-L1134) (and I guess that on appveyor the failure is likely the same).

it is related to prettier :-), it looks that it just wants a space between `{` and `gecko`.

Reply:
https://github.com/mozilla/addons-linter/pull/2285/commits/0f73e1805aaca5a9f1dfb1f9d3214a7ff0a922f9